packages/edit-navigation/src/components/header/style.scss

```diff
@@ -44,11 +44,57 @@
 	}
 }
 
-.edit-navigation-header__subtitle {
-	display: block;
-	margin: 0;
-	font-size: 15px;
-	font-weight: normal;
+.edit-navigation-menu-actions {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+
+	.edit-navigation-menu-actions__subtitle-wrapper {
+		display: flex;
+		flex-direction: row;
+		justify-content: center;
+		align-items: center;
+	}
+
+	.edit-navigation-menu-actions__switcher-toggle {
+		padding: 0;
+		min-width: 0;
+	}
+}
+
+.edit-navigation-menu-actions__switcher-dropdown {
+	// Appear below the modal overlay.
+	z-index: z-index(".components-popover.edit-navigation-menu-actions__switcher-dropdown");
+
+	.components-menu-group.has-hidden-separator {
+		padding: 0;
+	}
```

Comment:
This caught my eye. What's it's purpose? Short comment would be great.

Reply:
This is a hack, so the "Create new menu" button can take full width of the MenuGroup.

I will add a comment, thanks for the reminder.

Reply:
Done ✅